rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -171,6 +171,8 @@ bool LuaUnsyncedCtrl::PushEntries(lua_State* L)
 	REGISTER_LUA_CFUNC(AddWorldUnit);
 
 	REGISTER_LUA_CFUNC(DrawUnitCommands);
+	REGISTER_LUA_CFUNC(AddBuildSquare);
+	REGISTER_LUA_CFUNC(RemoveBuildSquare);
```

Comment:
I think it's a good foundation from the API cleanness perspective, but perhaps add `Unit` in between `{Add,Remove}` and `Build`?

Reply:
done